openshift/templates/tools/pipeline.yaml

```diff
@@ -40,14 +39,11 @@ objects:
   status:
     lastVersion: 0
 parameters:
-- name: "APP_NAME"
-  displayName: "Application Name"
-  description: "Name of the application this pipeline will build and deploy."
-  required: true
-- name: "COMP_NAME"
-  displayName: "Component Name"
-  description: "Name of the application component this pipeline will build and deploy."
+- name: "NAME"
+  displayName: "Pipeline Name (without -pipeline suffix)"
+  description: "Name of the pipeline, e.g cicd, promote-to-test, sonarqube. A -pipeline suffix will be appended to the name provided."
   required: true
+  value: cicd
 - name: "GITHUB_WEBHOOK_SECRET"
   displayName: "GitHub Webhook Secret"
```

Comment:
Probably should generate the secret for the pipeline by default.

Reply:
It DOES generate  secret by default. See the lines below. It also allows a user to supply their own secret (if they want to and are aware of risks)

```
  generate: "expression"
  from: "[a-zA-Z0-9]{40}"
```